tests/src/test/java/org/sonarqube/tests/issue/IssueNotificationsTest.java

```diff
@@ -156,7 +156,7 @@ public void notifications_for_new_issues_and_issue_changes() throws Exception {
     assertThat((String) message.getContent()).contains("Severity");
     assertThat((String) message.getContent()).contains("One Issue Per Line (xoo): 17");
     assertThat((String) message.getContent()).contains(
-      "See it in SonarQube: http://localhost:9000/project/issues?id=sample&createdAt=2015-12-15T00%3A00%3A00%2B");
+      "See it in SonarQube: http://localhost:9000/project/issues?id=sample&branch=master&createdAt=2015-12-15T00%3A00%3A00%2B");
```

Comment:
@julienlancelot I find it strange that `&branch=master` is needed here *but* not in the assertion on line 167 a few lines below here. I'm investigating now

Reply:
I looked but I cannot find the reason for this different behavior :-/